app/models/container_component_status.rb

```diff
@@ -1,3 +1,9 @@
 class ContainerComponentStatus < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => "ems_id"
+
+  virtual_column :name,         :type => :string
```

Comment:
@blomquisg 
Hi Greg can you have someone from provider team review the app/models changes.